packages/strapi-design-system/src/v2/Breadcrumbs/Breadcrumbs.js

```diff
@@ -38,11 +35,9 @@ export const Breadcrumbs = ({ label, children, ...props }) => {
   );
 };
 
-const crumbType = PropTypes.shape({ type: PropTypes.oneOf([Crumb, CrumbLink, CrumbSimpleMenu]) });
-
 Breadcrumbs.displayName = 'Breadcrumbs';
 
 Breadcrumbs.propTypes = {
-  children: PropTypes.oneOfType([PropTypes.arrayOf(crumbType), crumbType]).isRequired,
+  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
```

Comment:
I'm wondering if the initial definition was wrong and if this would be a better and working fix:

```suggestion
  children: PropTypes.arrayOf(PropTypes.oneOfType([Crumb, CrumbLink, CrumbSimpleMenu])).isRequired,
```

We should be able to make it work without opening it to `node`, as it works for the `SimpleMenu` too.

Reply:
I explained this in a comment you can see before your review ⬆️
I first tried the 
`children: PropTypes.arrayOf(PropTypes.oneOfType([Crumb, CrumbLink, CrumbSimpleMenu])).isRequired,`
solution but it doesn't help us in some cases like for ML Breadcrumbs
in there I created a CrumbSimpleMenuAsync component which returns CrumbSimpleMenu - when doing that we still have proptypes error since CrumbSimpleMenuAsync is not a valid component 🤷🏻‍♀️
but I agree I'm not a fan of just opening it to node - but if not we'll have proptypes errors 🥲

while checking the comments on my code I've written before your review could you also check this one because we still have a proptype error here 🙏 
<img width="985" alt="image" src="https://user-images.githubusercontent.com/71838159/181028517-32e5588c-1876-4fa8-8253-bdcf100c5add.png">

Reply:
> I tried to allow both version but since I have to rename at least one MenuItem component with as it still considers it as a different component

I think this might be the issue. Maybe we need another wrapper then, so that `Crumb`, `CrumbLink` and `CrumbSimpleMenu` are always a `div` (e.g.) and the `as` prop is applied to their children, so that we never change the way the actual container is rendered?

Reply:
I'm sorry I don't understand 🙈
MenuItem and Breadcrumbs don't really have the same issue I think.
If we keep saying to Breadcrumbs that it can only allow Crumb, CrumbLink, CrumbSimpleMenu components, then it will keep sending error if we need to add more logic as for CrumbSimpleMenuAsync no?

Reply:
What I mean is: instead of applying the `as` prop on the top-level DOM node you could try to apply it on a child instead. E.g.

```js
const CrumbLink = (props) => <div><Box {...props} /></div>
```

I'm wondering if the `as` prop creates the issue here, so that the actual components are not recognized by the prop-types.

Reply:
okay I see what you mean but I don't think it will fix the CrumbSimpleMenuAsync situation? 🤔
when I use CrumbSimpleMenuAsync I don't use as prop so I don't think that is the issue, except if I'm missing a piece of knowledge there? 

what I need to do is:
```
const CrumbSimpleMenuAsync = () => {
  // stuff happening here
  return <CrumbSimpleMenu />
}
```

and then:
```
const BreadcrumbsInML = () => {
  return 
  <Breadcrumbs>
    <CrumbSimpleMenuAsync />
  </Breadcrumbs>
}
```

without having Breadcrumbs complaining because I passed a children that is not CrumbSimpleMenu, CrumbLink or Crumb - are we talking about the same thing? 👀

Reply:
@ronronscelestes I'm also guessing here, but there must be a solution to this problem other than opening up the type to `node` :) Let me do some research ...

Reply:
note: we did some pair-programming on this issue and didn't find a working solution - we will keep `node` type for now